services/licenses.js

```diff
@@ -0,0 +1,33 @@
+const assert = require('assert');
+
+function normalizeTemplate(template) {
+  const { title } = template;
+  return title.replace(/^Template:/, '');
+}
+
+function formatPageTemplates(response) {
```

Comment:
same thing, I would propose the name `formattedTemplateNames(pageTemplates)`, what do you think?